plugins/bcc-login/includes/class-bcc-login-settings.php

```diff
@@ -427,70 +428,27 @@ function register_settings() {
                 $this->options_page,
                 'groups',
                 array(
-                    'name' => 'bcc_full_content_access_groups',
-                    'value' => join(",", $this->_settings->full_content_access_groups),
+                    'targetGroupsName' => 'bcc_full_content_access_groups',
```

Comment:
full content access is more like a permanent visibility group than a target group.

Reply:
what about just using "groupName" then?